frontend/assets/svgs/compressed/access.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><svg viewBox="0 0 123 86" fill="none" xmlns="http://www.w3.org/2000/svg" width="123" height="86"><path fill-rule="evenodd" clip-rule="evenodd" d="M25.72 7.88a2.46 2.46 0 012.43 2.44c0 1.33-1.1 2.45-2.43 2.45a2.45 2.45 0 010-4.9z" fill="#BFD4C3"/><path fill-rule="evenodd" clip-rule="evenodd" d="M17.05 7.88a2.45 2.45 0 010 4.89 2.45 2.45 0 01-2.43-2.45c0-1.34 1.1-2.44 2.43-2.44z" fill="#FFCEA8"/><path fill-rule="evenodd" clip-rule="evenodd" d="M8.38 7.88a2.45 2.45 0 010 4.89 2.45 2.45 0 01-2.43-2.45c0-1.34 1.1-2.44 2.43-2.44z" fill="#EBB3B3"/><path fill-rule="evenodd" clip-rule="evenodd" d="M101.19 0c4.84 0 8.81 4 8.81 8.87v54.27a8.88 8.88 0 01-8.81 8.88H8.8c-4.84 0-8.8-4-8.8-8.88V8.87A8.87 8.87 0 018.8 0h92.39zm0 2.58H8.8a6.27 6.27 0 00-6.24 6.29v8.48h104.88V8.87c0-3.46-2.8-6.3-6.25-6.3zm6.25 17.35H2.56v43.21c0 3.48 2.8 6.3 6.24 6.3h92.39a6.29 6.29 0 006.25-6.3V19.93z" fill="#0288D1"/><path d="M121.75 66.55a17.65 17.65 0 11-35.3 0 17.65 17.65 0 0135.3 0z" fill="#fff" stroke="#0288D1" stroke-width="2"/><path d="M113.16 75.6l-.1-3.6-6.77-6.8a5.79 5.79 0 10-11.24-1.9 5.8 5.8 0 007.35 5.6l1.45 1.46h2.86v2.47h3.18v2.78h3.27zM99.4 63.67c-1 0-1.8-.8-1.8-1.8s.8-1.8 1.8-1.8c.99 0 1.79.8 1.79 1.8s-.8 1.8-1.8 1.8z" fill="#0288D1"/></svg>
```

Comment:
There's something odd here. these icons inside compressed folder should be discard. The icons are directly inside `/assets/svgs/`